packages/prettier-config/README.md

```diff
@@ -0,0 +1,14 @@
+# `@shopify/prettier-config`
+
+[![Build Status](https://travis-ci.com/Shopify/web-foundation.svg?branch=master)](https://travis-ci.com/Shopify/web-foundation)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](../../LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Fprettier-config.svg)](https://badge.fury.io/js/%40shopify%2Fprettier-config.svg)  [![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/prettier-config.svg)](https://img.shields.io/bundlephobia/minzip/@shopify/prettier-config.svg) 
+
+Shared prettier configuration
+
+## Installation
+
+```bash
+$ yarn add @shopify/prettier-config
```

Comment:
Should this actually say `yarn add --dev @shopify/prettier-config` as the documentation for `@shopify/eslint-plugin` and `@shopify/stylelint-plugin` does?

Reply:
Yes, and also mention that it can be used by adding a `prettier` field in a project's `package.json`.